src/tuf/target.ts

```diff
@@ -27,7 +27,13 @@ export async function readTarget(
   return new Promise((resolve, reject) => {
     fs.readFile(path, 'utf-8', (err, data) => {
       if (err) {
-        reject(new InternalError(`error reading target: ${err}`));
+        reject(
+          new InternalError({
+            code: 'TUF_TARGET_READ_ERROR',
```

Comment:
Thoughts on using some kind of `namespace-prefix_verb/action_resource/thing_error` pattern in codes?
```suggestion
            code: 'TUF_READ_TARGET_ERROR',
```